server/middleware/office.validation.js

```diff
@@ -0,0 +1,33 @@
+const validateOffice = (req, res, next) => {
+  let verified = true;
+  const error = [];
+
+  const { type, name } = req.body;
+  const typesOfGov = ['Federal', 'State', 'Local'];
+
+  if (!type || !type.trim()) {
+    verified = false;
+    error.push({ type: 'type must be present' });
+  }
+
+  if (!typesOfGov.includes(type)) {
+    verified = false;
+    error.push({ type: 'please specify a valid type' });
```

Comment:
Can you make your error message more descriptive? what are the valid types --> `please specify a valid type of either 'Federal', 'State', 'Local'`